src/types.rs

```diff
@@ -443,6 +482,64 @@ impl ArrayType {
     }
 }
 
+/// A valid Astarte float.
+///
+/// It cannot be Nan, Infinite or subnormal.
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd)]
+#[repr(transparent)]
+pub struct Float(f64);
```

Comment:
Since it contains an f64, why don't call directly `Double`? `Double(Float),` it is little strange, I think

Reply:
Sure, make sense. I wanted to call it F64 but was it looked wrong so it became Float. But double make sense.